Deepanshu_Balani/project2 tictactoe/assets/tic-tac.js

```diff
@@ -0,0 +1,108 @@
+
+var N_SIZE = 3,
+  EMPTY = '&nbsp;',
+  boxes = [],
+  turn = 'X',
+  score,
+  moves;
+
+/*initialization of tic-tac-toe board*/
+function init() {
+  var board = document.createElement('table');
+  board.setAttribute('border', 1);
+  board.setAttribute('cellspacing', 0);
+
+  var identifier = 1;
+  for (var i = 0; i < N_SIZE; i++) {
+    var row = document.createElement('tr');
+    board.appendChild(row);
+    for (var j = 0; j < N_SIZE; j++) {
+      var cell = document.createElement('td');
+      cell.setAttribute('height', 120);
+      cell.setAttribute('width', 120);
+      cell.setAttribute('align', 'center');
+      cell.setAttribute('valign', 'center');
+      cell.classList.add('col' + j, 'row' + i);
+      if (i == j) {
+        cell.classList.add('diagonal0');
+      }
+      if (j == N_SIZE - i - 1) {
+        cell.classList.add('diagonal1');
+      }
+      cell.identifier = identifier;
+      cell.addEventListener('click', set);
```

Comment:
This will create memory leaks, get rid of this issue

Reply:
removeEventlistener() added.